src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java

```diff
@@ -41,19 +37,6 @@
     private String hearingBookedRepsEmailId;
     @Value("${notification.hearingBooked.representative.smsId}")
     private String hearingBookedRepsSmsId;
-
-    @Value("${notification.evidenceReceived.appellant.emailId}")
-    private String evidenceReceivedAppellantEmailId;
-    @Value("${notification.evidenceReceived.appellant.smsId}")
-    private String evidenceReceivedAppellantSmsId;
-    @Value("${notification.paper.evidenceReceived.representative.emailId}")
-    private String paperEvidenceReceivedRepsEmailId;
-    @Value("${notification.evidenceReceived.representative.smsId}")
-    private String paperEvidenceReceivedRepsSmsId;
-    @Value("${notification.oral.evidenceReceived.representative.emailId}")
-    private String oralEvidenceReceivedRepsEmailId;
-    @Value("${notification.evidenceReceived.representative.smsId}")
-    private String oralEvidenceReceivedRepsSmsId;
```

Comment:
There are also duplicate lines for evidenceReceived in NotificationsFunctionalTest

Reply:
The lines are removed.